src/styles/main.scss

```diff
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: calc((#{$day-size} * 7) + (#{$gap} * 6));
+}
+  @for $i from 1 through 31 {
+    .calendar__day:nth-child(#{$i})::before {
+      content: "#{$i}";
+    }
+  }
+
+  .month-length--start-day .calendar__day:first-child {
+    margin-left: $calc-margin;
```

Comment:
Ensure that the variable `$calc-margin` is defined. It is used here and needs to be set for the SCSS to compile correctly.